docs/integration_listen_events.md

```diff
@@ -0,0 +1,119 @@
+| async_track_state_change_event     | Track specific state change events indexed by entity_id
+| async_track_state_added_domain     | Track state change events when an entity is added to domains
+| async_track_state_removed_domain   | Track state change events when an entity is removed from domains
+| async_track_state_change_filtered  | Track state changes with a TrackStates filter that can be updated
+| async_track_same_state             | Track the state of entities for a period and run an action
+
+### Tracking template changes
+
+| Function                           | Use case
+| ---------------------------------- | --------------------------------------------------------------------------
+| async_track_template               | Add a listener that fires when a a template evaluates to 'true'
+| async_track_template_result        | Add a listener that fires when the result of a template changes
+
+### Tracking entity registry changes
+
+| Function                                  | Use case
+| ----------------------------------------- | --------------------------------------------------------------------------
+| async_track_entity_registry_updated_event | Track specific entity registry updated events indexed by entity_id
+
+### Tracking time changes
+
+| Function                                  | Use case
+| ----------------------------------------- | --------------------------------------------------------------------------
+| async_track_point_in_time                 | Add a listener that fires once after a specific point in time
+| async_track_point_in_utc_time             | Add a listener that fires once after a specific point in UTC time
+| async_call_later                          | Add a listener that is called with a delay
+| async_track_time_interval                 | Add a listener that fires repetitively at every timedelta interval
+| async_track_utc_time_change               | Add a listener that will fire if time matches a pattern
+| async_track_time_change                   | Add a listener that will fire if local time matches a pattern
+
+### Tracking the sun
+
+| Function                                  | Use case
+| ----------------------------------------- | --------------------------------------------------------------------------
+| async_track_sunrise                       | Add a listener that will fire a specified offset from sunrise daily
+| async_track_sunset                        | Add a listener that will fire a specified offset from sunset daily
+
+## Listening to the event bus directly
+
+There are two functions available to create listeners. Both functions return a callable that cancels the listener. 
+
+- `async_listen_once` - Listen once for the event and never fire again
+- `async_listen` - Listen until canceled
+
+Its a rare case that `async_listen` is use since `EVENT_HOMEASSISTANT_START`, `EVENT_HOMEASSISTANT_STARTED`, and `EVENT_HOMEASSISTANT_STOP` are only ever fired once per run.
```

Comment:
```suggestion
It's a rare case that `async_listen` is used since `EVENT_HOMEASSISTANT_START`, `EVENT_HOMEASSISTANT_STARTED`, and `EVENT_HOMEASSISTANT_STOP` are only ever fired once per run.
```

Reply:
Thanks